.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!-- Please review the following before submitting a PR:

```

Comment:
Add somewhere 
" ### Status
Write status of your pull request: Finished/WIP (Note that your changes should be ready before creating new pull request)"